.github/workflows/builder_go_slsa3.yml

```diff
@@ -343,6 +343,7 @@ jobs:
           UNTRUSTED_ENV: "${{ needs.build-dry.outputs.go-env }}"
           UNTRUSTED_WORKING_DIR: "${{ needs.build-dry.outputs.go-working-dir }}"
           GITHUB_CONTEXT: "${{ toJSON(github) }}"
+          VARS_CONTEXT: "${{ toJSON(vars) }}"
```

Comment:
I forgot for the Go builder, vars can't affect the build so we probably shouldn't record them in this case.